exporter/containerimage/attestations.go

```diff
@@ -122,7 +122,7 @@ func supplementSBOM(ctx context.Context, s session.Group, target cache.Immutable
 }
 
 func decodeSPDX(dt []byte) (s *spdx.Document, err error) {
-	doc, err := spdx_json.Load2_3(bytes.NewReader(dt))
+	doc, err := spdx_json.Read(bytes.NewReader(dt))
```

Comment:
This is why `TestSBOMSupplements`.
spdk_json.Read does some nasty stuff to try and convert document formats, it seems there's a bug in the implementation.

Reply:
Probably somewhere in here: https://github.com/anchore/go-struct-converter/blob/main/converter.go#L43
And there's very little testing.

Reply:
Thanks @cpuguy83! updated with json unmarshal for now. Let me know if this is fine.

Reply:
Ah, I worked out what the issue with this was :cry:

Turns out, `SPDXVersion` is a required property, and the `TestSBOMSupplements` test just wasn't setting it :cry: